src/v3/src/transformer/i18n/transform.ts

```diff
@@ -117,8 +118,39 @@ export const transformAdditionalPhoneUITranslations = (formBag: FormBag) => {
   return formBag;
 };
 
+export const transformPasswordTooltipTranslations = (formBag: FormBag) => {
+  const { uischema } = formBag;
+
+  const traverseOptions = {
+    layout: uischema,
+    predicate: (el) => (el.options.inputMeta.secret === true),
+    callback: (el) => {
+      // eslint-disable-next-line
```

Comment:
it's best practice most of the time to put the rule name that you're targeting in the eslint-disable statements like this:
```suggestion
      // eslint-disable-next-line no-param-reassign
```